### PR TITLE
Prevent slider from overlapping

### LINKED ISSUE #241: Don't allow population sliders to reach the same point
It's meaningless for the population sliders to both be set to the same value, like 10m and 10m. That would mean a city must have a population exactly matching the value, per our predicate function:

https://github.com/ParkingReformNetwork/reform-map/blob/27fb01e5c3a4493b9500c709e0508e8cb8359893/src/js/populationSlider.ts#L159-L169

It also results in a bad user experience where the two icons overlap:

Captura de pantalla 2023-09-14 a la(s) 10 04 03 p m

### LINKED ISSUE #249: Prevent right population slider from sliding half-intervals

### PR DESCRIPTION
**Pull request by tunglinn**
Resolves #241 
Resolves #249 

By simplifying slider render, so the sliders are joined down the middle.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,7 +129,6 @@ <h3>Concept and Initial Report</h3>
           type="range"
           step="0.5"
           min="0"
-          value="0"
```
Comment (tunglinn) [inline index.html:132]: This is going to be set in draw().
Also by setting it early, it makes debugging with developer tools a little harder to read.

File: src/js/populationSlider.ts [hunk 2]
```diff
@@ -13,28 +13,26 @@ const draw = (
...
 
   sliders.left.setAttribute("max", newLeftMax.toString());
   sliders.right.setAttribute("min", newRightMin.toString());
+  sliders.left.setAttribute("value", leftIndex.toString());
+  sliders.right.setAttribute("value", rightIndex.toString());
 
   const intervalSizePx =
-    sliders.controls.offsetWidth / POPULATION_INTERVALS.length;
+    (sliders.controls.offsetWidth + THUMBSIZE / 2) /
+    POPULATION_INTERVALS.length;
```
Comment (tunglinn) [inline src/js/populationSlider.ts:27]: `THUMBSIZE / 2` is a guess. The slider wasn't taking up the whole div, and it looked like it was off by half a header.

File: src/js/populationSlider.ts [hunk 3]
```diff
@@ -13,28 +13,26 @@ const draw = (
...
   sliders.right.style.width = `${rightWidth + THUMBSIZE}px`;
 
   // The left slider has a fixed anchor. However, the right slider has to move
   // everytime the range of the slider changes.
-  const offset = 5;
-  sliders.left.style.left = `${offset}px`;
-  sliders.right.style.left = extend
-    ? `${leftWidth - intervalSizePx / 2 + offset}px`
-    : `${leftWidth + offset}px`;
+  sliders.right.style.left = `${leftWidth}px`;
```
Comment (tunglinn) [inline src/js/populationSlider.ts:35]: When we set `left` for the left slider, it doesn't respond to padding and margin attributes.
And we don't really need an offset.

File: src/js/types.ts [hunk 4]
```diff
@@ -59,8 +59,9 @@ export class PopulationSliders {
 
   /** Return the [leftIndex, rightIndex] of the sliders. */
   getCurrentIndexes(): [number, number] {
-    const get = (slider: HTMLInputElement): number =>
-      Math.floor(parseFloat(slider.value));
-    return [get(this.left), get(this.right)];
+    return [
+      Math.floor(parseFloat(this.left.value)),
+      Math.ceil(parseFloat(this.right.value)),
+    ];
```
Comment (tunglinn) [inline src/js/types.ts:65]: Main mechanism that prevents overlapping (along with the new simplified slider mech).

**Review by Eric-Arellano — APPROVED** [review]
Yay, thanks! Good job reviewing your own code!

**Pull request merged**